static/css/components/form.olform.less

```diff
@@ -48,10 +48,11 @@
     width: 4em;
   }
 
-  input[type=email],
-  input[type=password],
   .required {
-    padding: 8px;
+    border-style: solid;
+    border-width: 2px;
+    border-color: red;
```

Comment:
Consider accessibility concerns here, "border-color: red" will not be particularly visible to people with various forms of red-green colour blindness. The aria-required attribute should also be used for required fields, to make these more accessible.

Reply:
Thank you for calling attention to this. I added the aria-required property to the elements which I touched. I do not have significant experience in UI accessibility. After a brief research, my proposed solution would be to add a "* Required" string next to the input fields. However, I found implementing this cumbersome in the current jQuery-based page. I also think that implementing such would go beyond the scope of the issue, and my solution does not make accessibility worse. Therefore, I recommend that we channel your note regarding the color into e.g. [this](https://github.com/internetarchive/openlibrary/projects/29) or [this](https://github.com/internetarchive/openlibrary/projects/10) project.